data/json/items/melee/fake.json

```diff
@@ -19,6 +19,7 @@
     "id": "bio_blade_weapon",
     "name": { "str": "monomolecular blade" },
     "description": "A foot-long blade made from high-tech alloy and edged with bonded nanocrystals.",
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "good" },
```

Comment:
I'm not sure that the balance on a blade coming out of your arm would be good.  You'd lose a lot of the range of motion granted by a wrist.

Reply:
Both of these weapons are at neutral balance at best, since they don't take advantage of wrist or hand flexibility.

Reply:
Ok, i guess that's fine.

For reference, i was following the guidelines that mark combat knives (Like the monoblade) as well balanced.

I also thought that unarmed weapons like the claws were all considered good balanced, but neutral works too.